students/arkadiusz_kasprzyk/lesson_06_dicts/uppercase.py

```diff
@@ -0,0 +1,91 @@
+# uppercase.py
+
+
+def capitalize_word(word):
+    """
+    Parameters
+    ----------
+    word: str
+
+    Returns
+    -------
+    word_cap: str
+        First letter of `word` turned to uppercase (if it's a letter).
+
+    Examples
+    --------
+    capitalize_word("spam")         # 'Spam'
+    capitalize_word("Spam")         # 'Spam'
+    capitalize_word("_spam")        # '_spam'
+    capitalize_word(" spam")        # ' spam'
+    capitalize_word("_spam eggs")   # '_spam eggs'
+    """
+    word0 = word[0]
+    char0 = ord(word0)
+    if char0 in range(97, 123):
+        word0 = chr(char0 - 32)
+
+    word_cap = word0 + word[1:]
+    return word_cap
+
+
+def capitalize(text):
+    """
+    Parameters
+    ----------
+    text: str
+
+    Returns
+    -------
+    text_cap: str
+        Each word of `txt` capitalized.
+
+    Examples
+    --------
+    capitalize("spam and eggs")     # 'Spam And Eggs'
```

Comment:
Did you know that you can actually run tests from the documentation. There is even a pytest plugin for that: https://docs.pytest.org/en/latest/doctest.html

Reply:
I know about but don't know details yet. Sure it's simple but... to many "simple" things to grasp :) Thanks for reminding!